WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/widget/configuration/DataTypeSelectionDialogFragment.kt

```diff
@@ -0,0 +1,73 @@
+package org.wordpress.android.ui.stats.refresh.lists.widget.configuration
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import android.widget.RadioGroup
+import androidx.appcompat.app.AppCompatDialogFragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import dagger.android.support.AndroidSupportInjection
+import org.wordpress.android.R
+import org.wordpress.android.ui.stats.refresh.lists.widget.configuration.StatsDataTypeSelectionViewModel.DataType
+import org.wordpress.android.ui.stats.refresh.lists.widget.configuration.StatsDataTypeSelectionViewModel.DataType.COMMENTS
+import org.wordpress.android.ui.stats.refresh.lists.widget.configuration.StatsDataTypeSelectionViewModel.DataType.LIKES
+import org.wordpress.android.ui.stats.refresh.lists.widget.configuration.StatsDataTypeSelectionViewModel.DataType.VIEWS
+import org.wordpress.android.ui.stats.refresh.lists.widget.configuration.StatsDataTypeSelectionViewModel.DataType.VISITORS
+import org.wordpress.android.util.image.ImageManager
+import javax.inject.Inject
+
+class DataTypeSelectionDialogFragment : AppCompatDialogFragment() {
```

Comment:
Wonder if we should rename this (and the color picker) one to `StatsWidgetDataTypeSelectionDialogFragment` to stay consistent with a general naming scheme and 'StatsWidgetSiteSelectionDialogFragment'? Either option is fine with me :)

Reply:
makes sense 👍 , I'll remove both